operations/resiliency/NTP_Resiliency.md

```diff
@@ -28,35 +28,7 @@ This procedure requires administrative privileges.
 
     ```bash
     ncn-m002# for i in ncn-{w,s}00{1..3} ncn-m00{2..3}; do echo \
-    "------$i--------"; ssh $i '/srv/cray/scripts/metal/set-ntp-config.sh'; done
+    "------$i--------"; ssh $i '/srv/cray/scripts/metal/csm_ntp.py'; done
```

Comment:
`.../scripts/common/chrony/csm_ntp.py` is where the script ended up. We either need to move it or adjust this line.

Reply:
Fixed